### DISCUSSION TITLE
Emoji isn't colored in commits page

### DISCUSSION THREAD

**Discussion by plainheart** [community/community#8252] [Other Feature Feedback, Questions, & Ideas] [answered]
**UPDATED:** It seems to be working in the latest Chrome.

---

I noticed the emoji has no color and looks bigger than the one on the repository/branch page.

- https://github.com/apache/echarts-handbook/tree/gh-pages (Colored)

<img src="https://user-images.githubusercontent.com/26999792/143998186-6a826a75-4ef9-41ed-958e-fc0603fddc99.png" width="600">

- https://github.com/apache/echarts-handbook/commits/gh-pages (Uncolored)

<img src="https://user-images.githubusercontent.com/26999792/143998073-b7a6235d-a7dc-489f-9e28-43345d371b88.png" width="600">

Is it expected behavior?

Regards.

**Comment by Katsute** [comment, marked as answer]
This is an issue with chromium, not GitHub https://bugs.chromium.org/p/chromium/issues/detail?id=1266022

**Reply by plainheart** [reply]
Thanks for your reply. It's interesting. I looked through the bug report detail, it can work if we remove the `text-bold` class from the `a` element on the commits page.
